frontend/set_build_variables.js

```diff
@@ -3,7 +3,7 @@ replace = require("replace-in-file");
 
 var replacements = [];
 var envKeys = Object.keys(envVars);
-
+var environment = process.env.ENV || 'develop';
```

Comment:
maybe this will never really be undefined, but there's no `develop` environment file for that fallback, I assume that would error

Reply:
right, setting default to qa